frontend/pages/index.tsx

```diff
@@ -40,7 +40,7 @@ const HomePage = () => {
     const response = await requestWithoutAuthentication('POST', 'waitlist_entry/', { email });
 
     if (response) {
-      if (response.status === 201) {
+      if (response.status === 200 || response.status === 201) {
```

Comment:
If you change the API as suggested then this change will not be necessary.

Reply:
I removed the 200 HTTP response condition.